src/com/lftechnology/java/training/sanish/application/utility/UserInput.java

```diff
@@ -0,0 +1,172 @@
+package com.lftechnology.java.training.sanish.application.utility;
+
+import com.lftechnology.java.training.sanish.application.component.Constants;
+
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Provide methods to get inputs form console
+ *
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class UserInput {
+    private static final Logger LOGGER = Logger.getLogger(UserInput.class.getName());
+
+    private UserInput() {
+    }
+
+    /**
+     * Scan number from console and return number
+     *
+     * @param {@link Scanner}
+     * @param {@link Integer} Minimum value acceptable
+     * @param {@link Integer} Maximum value acceptable
+     * @return {@link Integer}
+     * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+     */
+    public static int getIntegerNumber(Scanner scanner, int min, int max) {
+        int intNum = 0;
+        boolean checkNextInput = true;
+        while (checkNextInput) {
+            if (!scanner.hasNextInt()) {
+                LOGGER.log(Level.WARNING, Constants.ENTER_VALID_NUMBER_MSG);
+                scanner.next();
+            } else {
+                intNum = scanner.nextInt();
+                if (min != -1 && max != -1) {
+                    if (intNum < min || intNum > max) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS134)